src/components/Input/Input.js

```diff
@@ -57,9 +57,11 @@ const Input = forwardRef(
             aria-labelledby={label ? labelId : null}
             ref={ref}
           />
-          <FieldErrorText role="alert" aria-invalid={errorMessage !== null}>
-            {errorMessage}
-          </FieldErrorText>
+          {errorMessage !== null && (
```

Comment:
Write a proper test case for this change.

Reply:
All test cases are already in place. Updated snapshot as per the change.

Reply:
Previously only `aria-invalid` was depend on `errorMessage` now complete `FieldErrorText` component dependent on `errorMessage`. So the test case should be updated accordingly.

Reply:
As i said. Required test case is already in place.
```
it("renders input with error", () => {
    const { container } = renderInputComponent({
      errorMessage: "Something Went Wrong"
    });
    expect(container.firstChild).toMatchSnapshot();
  });
```
It even have 100% coverage. 
```
File                               |  % Stmts | % Branch |  % Funcs |  % Lines | 
Input.js                           |      100 |      100 |      100 |      100 |  
Input.styles.js                    |      100 |      100 |      100 |      100 |
```

Reply:
Okay got it. thanks for clarification